### ISSUE TITLE
Gradio is getting Time-Out

### ISSUE THREAD

**Issue by jorgenascimentoifs** [TheLastBen/fast-stable-diffusion#1638]
504 Gateway Time-out

**Comment by GoWithitRoger** [comment]
Same here. Worked 2 days ago without issue. No changed on my end.

**Comment by jorgenascimentoifs** [comment]
yeah. I didn't have any errors in google colab, only in gradio. let's see if anyone can help.

**Comment by jorgenascimentoifs** [comment]
![Captura de tela 2023-02-26 175838](https://user-images.githubusercontent.com/47475762/221436971-9048f808-6f4c-4d7f-8b9b-f8f25dcaf1f0.png)

**Comment by jorgenascimentoifs** [comment]
click on Use_localtunnel and it will work!

**Comment by Monkqp** [comment]
It's an issue with gradio.live. Any shared urls from that website aren't working. Not sure why

**Comment by timothywcrane** [comment]
Unable to upgrade extensions or run ANY actions on local tunnel, better than a blue/gray screen, but not much use ;( .

**Comment by jnyfil** [comment]
I struggled with this problem and tried to run [localtunnel-server](https://github.com/localtunnel/server) on my machine, but couldn't use it without a domain. 
So I created [localtunnel-loopback](https://github.com/jnyfil/localtunnel-loopback) to solve this problem. Additionally, I created the [sd-webui-localtunnel-client](https://github.com/jnyfil/sd-webui-localtunnel-client) extension to make it easy to use in the webui, so using the two together makes it easier, more reliable, and faster.
